src/etc/inc/openvpn.inc

```diff
@@ -447,6 +447,16 @@ function openvpn_get_digestlist() {
 	return $digests;
 }
 
+function openvpn_get_keydirlist() {
+	$keydirs = array(
+		'-1' => gettext('Use default keydir direction'),
```

Comment:
This value should probably be unset/empty `''` because otherwise upgraded configurations will never match any of these cases.